x-pack/plugin/core/template-resources/src/main/resources/ml/meta_index_mappings.json

```diff
@@ -36,6 +36,15 @@
       "end_time": {
         "type": "date"
       },
+      "skip_result": {
```

Comment:
The `.ml-meta` index is a managed system index. Please increment the mapping version at https://github.com/elastic/elasticsearch/blob/fe02cf0b2bb2ba9dcab89ee724c4a5481fa20ac4/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/MlMetaIndex.java#L20

Once the cluster is fully upgraded the mappings will be automatically updated

Reply:
I'll update this.